docs/api/ShallowWrapper/renderProp.md

```diff
@@ -0,0 +1,85 @@
+# `.renderProp(propName, ...args) => ShallowWrapper`
+
+Calls the current wrapper's property with name `propName` and the `args` provided.
+Returns the result in a new shallow wrapper.
+
+NOTE: can only be called on wrapper of a single non-DOM component element node.
```

Comment:
where is "non-DOM" being enforced? we might need to check the node type isn't "host" in `renderProp`.

Reply:
Nice catch. It wasn't actually enforced so far. I now added an explicit check and amended the last commit. Copied that one from `simulateError`.